AppLand/scss/main.scss

```diff
@@ -0,0 +1,49 @@
+/*========================================================================================
+*********   
+*******     IMPORT ALL OF THE STYLES HERE
+*****       
+***         
+*/
+
+    /*--------------------------------
+    ***  SET DIRECTION BEFORE ANYTHING
+    **   
+    */
+
+    @import "typography";
+        
+        
+    /*--------------------------------
+    ***  IMPORT GENERAL AND IMPORTANT
+    **   VARIABLES
+    */
+
+        @import "variables";
+
+    /*--------------------------------
+    ***  
+    **   appLand
+    */   
+
+    @import "appLand";
+
+    /*--------------------------------
+    ***  Buttons
+    **   
+    */
+
+    @import "button.scss";
+
+    /*--------------------------------
+    ***  shared files 
+    **  
+    */   
+
+    @import "../../shared/style.css";
```

Comment:
style.css or style.scss ?!

Reply:
I changed it to shared.css
Yes, this is a css file

Reply:
I know it's css I can read it, 
you import CSS in SCSS ?! do you think it's correct